### DISCUSSION TITLE
Basic video stream

### DISCUSSION THREAD

**Discussion by assoft** [remix-run/remix#3137] [General]
I wonder if you could make an example for a simple video stream with a remix? Please...

**Comment by sergiodxa** [comment]
You want to stream the video from a Remix resource route? I would recommend you to try to use a dedicated and specialized server instead, everything related to videos is complex

**Reply by kiliman** [reply]
Exactly, then that way the only thing you have to do with Remix is render a video component pointing to your streaming URL.

**Reply by assoft** [reply]
Yes, I know, but I don't want anything overly complicated. My only goal is to get rid of using an external api and meet this need with remix. (Only for experimental purposes locally)

Thank you!

**Reply by kiliman** [reply]
There's nothing preventing you from pointing the `<video src={videoUrl}/>`. That's what I do. I uploaded a teaser video to Cloudinary and simply play it back that way.
```ts
        <video
          controls
          poster={poster}
          src="https://res.cloudinary.com/.../video.mp4"
        ></video>
```

**Reply by assoft** [reply]
😊
